instaloader/instaloadercontext.py

```diff
@@ -447,6 +448,19 @@ def get_json(self, path: str, params: Dict[str, Any], host: str = 'www.instagram
                 response_headers.clear()
                 response_headers.update(resp.headers)
             if resp.status_code == 400:
+                # Raise AbortDownloadException in case of substantial Instagram requirements to stop producing more requests
+                content_type = resp.headers.get('Content-Type')
+                if 'application/json' in content_type or not content_type:
+                    try:
+                        resp_json = resp.json()
+                    except ValueError:
```

Comment:
resp.json() would actually raise json.decoder.JSONDecodeError. Though it is a ValueError, catching JSONDecodeError would make that part of the code more explanatory regarding what exactly is the error we're looking out for.